### ISSUE TITLE
print HTML not working

### ISSUE THREAD

**Issue by ahsan-ab** [InteractiveObject/StarIOPlugin#6]
how to pass HTML ?

**Comment by warbadev** [comment]
well u can just put ur Html into a string and then print that string
but what i really need is how print a logo
in epson i usually  make the printer to print the logo after cutting the recipt to prepare it for the next customer but here i played with the manual and every where but couldnt find a way to print the logo auto or manual with each recipt

**Comment by ahsan-ab** [comment]
yes i put html into string but its print without styling ... any idea how to print with styling ?

**Comment by warbadev** [comment]
u have to use some of commands it depends how u what u need but every thing u need is in
[here](http://www.starasia.com/%5CDownload%5CManual%5CStarPRNT_cm_en.pdf) u need to convert every thing into hex and move it to ur string 
its a bit tricky am still a newbie and still struggling with my logo on how to put it in my reciept
